.github/actions/common/xds-service/action.yml

```diff
@@ -0,0 +1,51 @@
+name: "xDS Service Common Test"
+description: "Auto common test for xds service"
+runs:
+  using: composite
+  steps:
+    - name: start minikube
+      uses: medyagh/setup-minikube@latest
+      id: minikube
+      with:
+        minikube-version: 1.31.2
+        driver: docker
+        container-runtime: docker
+    - name: Delay for 10 seconds to ready k8s
```

Comment:
It cannot be ensured that the preparation is complete within 10 seconds. Is there any other method to accurately determine the preparation?

Reply:
The delay is sufficient.